clients/python/aptible_client/helpers/misc.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+from logging import Logger
+from pathlib import Path
+
+from .constants import AUTH_API_URL, CLOUD_API_URL
+from ..api.assets_api import AssetsApi
+from ..api.environments_api import EnvironmentsApi
+from ..configuration import Configuration
+from ..model.environment_input import EnvironmentInput
+
+
+def get_client_configuration() -> Configuration:
+    """
+
+    :return:
+    """
+    local_token = os.getenv("APTIBLE_TOKEN")
+    if local_token is None:
+        try:
+            tokens_json = open(str(Path.home() / ".aptible" / "tokens.json"), "r").read()
+            local_token = json.loads(tokens_json).get(AUTH_API_URL)
+        except Exception:
+            print("Unable to retrieve tokens from aptible client's default location")
```

Comment:
We should be using logging statements, not print statements, for library code. The only time we should really use print is in the typer command functions themselves if we have any.

I'd also add more data to this- like `run aptible login to create your token file`.

We should also throw a more specific exception so it can be caught.

Reply:
yeah i can switch to using a logger, the print statements/warnings and documentation improvements are coming with @NJalal7 pair as this PR is already large

Reply:
<img width="763" alt="image" src="https://user-images.githubusercontent.com/2961973/195118295-abc2a728-5486-4cf5-a541-203dba5b8152.png">

yeah itll raise this error so someone can still catch that more specific one if they desire

Reply:
this will return a file not found error:

<img width="1282" alt="image" src="https://user-images.githubusercontent.com/2961973/195118475-affae3a4-0714-483b-b53c-e1c98fc64da5.png">

updated to reference code with requested update